templates/community/mission.html

```diff
@@ -60,7 +60,7 @@ <h3>What is open source?</h3>
         <li>The licence must not restrict other software.</li>
         <li>The licence must be technology-neutral.</li>
       </ol>
-      <p><a class="p-link--external" href="http://www.opensource.org/docs/definition.php">Take a look at the Open Source Initiative's definition of open-source software for a more detailed explanation</a>.</p>
+      <p><a class="p-link--external" href="https://www.opensource.org/docs/definition.php">Take a look at the Open Source Initiative's definition of open-source software for a more detailed explanation</a>.</p>
```

Comment:
The final destination of this link is https://opensource.org/docs/definition.php